lib/WorkerHandler.js

```diff
@@ -157,11 +188,11 @@ function WorkerHandler(script, options) {
     me.terminating = false;
 
     for (var id in me.processing) {
-      if (me.processing.hasOwnProperty(id)) {
+      if (me.processing[id] !== undefined) {
```

Comment:
`me.processing` is an `Object.create(null)` so both does not need `hasOwnProperty` and does not support `hasOwnProperty`. So it can simply become `!== undefined`.